src/client/Microsoft.Identity.Client/Internal/Constants.cs

```diff
@@ -25,6 +25,8 @@ internal static class Constants
         public const string OrganizationsTenant = "organizations";
         public const string CommonTenant = "common";
 
+        public const string MsaDomainName = "live.com";
```

Comment:
Let's rename this constant as MSA really has a ton of domain names (outlook.com, hotmail.com etc.). This is `UserRealmMsaDomainName` or smth similar.

Reply:
done. thanks. nice idea.

---
In reply to: [530299423](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2242#discussion_r530299423) [](ancestors = 530299423)